models/points.go

```diff
@@ -281,8 +281,8 @@ func ParseKeyBytes(buf []byte) ([]byte, Tags) {
 	return buf[:i], tags
 }
 
-func ParseTags(buf []byte) (Tags, error) {
-	return parseTags(buf), nil
+func ParseTags(buf []byte) Tags {
```

Comment:
the `error` return value was always `nil`, so this removes a stack push / pop for every `ParseTags` call